docs/DeveloperGuide.md

```diff
@@ -148,6 +148,47 @@ The `Storage` component,
 
 Classes used by multiple components are in the `seedu.addressbook.commons` package.
 
+### Entity specific classes
+
+Yellowbooks has 3 primary entities: 'Person', 'Task', and 'Tag', these are represented by classes with the respective names. The 'Tag' class provides an association between a 'Person' and 'Task'.
```

Comment:
Instead of an association between Person and Task, maybe you should say a label for either a Person or a Task?